docs/operation-guide.md

```diff
@@ -0,0 +1,135 @@
+
+## Deploy TiDB cluster
+
+After TiDB Operator and Helm are deployed correctly, TiDB cluster can be deployed using following command:
+
+```shell
+$ helm install charts/tidb-cluster --name=${releaseName} --namespace=${namespace}
+$ kubectl get po -n ${namespace} -l app.kubernetes.io/name=tidb-operator
+```
+
+For customized deployment, you can modify `values.yaml` before installing the charts. Most of the variables are self-explanatory with comments.
+
+## Access TiDB cluster
+
+By default TiDB service is exposed using [`NodePort`](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport). You can modify it to `ClusterIP` which will disable access from outside of the cluster. Or modify it to [`LoadBalancer`](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) if the underlining Kubernetes supports this kind of service.
+
+By default TiDB cluster is deployed with a random generated password. You can specify a password by setting `tidb.password` in `values.yaml` before deploying. Whether you specify the password or not, you can retrieve the password through `Secret`:
+
+```shell
+$ PASSWORD=$(kubectl get secret -n ${namespace} ${clusterName}-tidb -ojsonpath="{.data.password}" base64 -c | awk '{print $6}')
+$ echo ${PASSWORD}
+$ kubectl get svc -n ${namespace} # check the available services
+```
+
+* Access inside of the Kubernetes cluster
+
+    When your application is deployed in the same Kubernetes cluster, you can access TiDB via domain name `demo-tidb.tidb.svc` with port `4000`. Here `demo` is the `clusterName` which can be modified in `values.yaml`. And the latter `tidb` is the namespace you specified when using `helm install` to deploy TiDB cluster.
+
+* Access outside of the Kubernetes cluster
+
+    * Using kubectl portforward
+
+        ```shell
+        $ kubectl port-forward -n ${namespace} svc/${clusterName}-tidb 4000:4000 &>/tmp/portforward-tidb.log
+        $ mysql -h 127.0.0.1 -P 4000 -u root -p
+        ```
+
+    * Using LoadBalancer
+
+        When you set `tidb.service.type` to `LoadBalancer` and the underlining Kubernetes support LoadBalancer, then a LoadBalancer will be created for TiDB service. You can access it via the external IP with port `4000`. Some cloud platforms support internal load balancer via service annotations, for example you can add annotation `cloud.google.com/load-balancer-type: Internal` to `tidb.service.annotations` to create an internal load balancer for TiDB on GKE.
+
+    * Using NodePort
+
+        You can access TiDB via any node's IP with tidb service node port. The node port is the port after `4000`, usually greater than `30000`.
+
+## Scale TiDB cluster
+
+TiDB Operator has full support of horizontal scaling. But for vertical scaling, if you're using local volumes for PD and TiKV, then scaling up may cause pod pending if the node doesn't have enough resources. So it's not recommended to do vertical scaling.
```

Comment:
What about vertical scale-down?

Reply:
What about TiDB SQL? I should be able to scale those vertically?

I created a github issue focused on TiKV/PD here: https://github.com/pingcap/tidb-operator/issues/191

Reply:
I've added the recommendations for TiDB cluster deployment.

Reply:
A lot more thorough now, thanks! I still don't know from the guide if I can safely vertical scale TIDB SQL.